src/common/sphereversion.h

```diff
@@ -46,6 +46,9 @@
 	#else
 		#define SPHERE_VER_FILEOS_STR		"[FreeBSD-32]"
 	#endif
+#elif __APPLE__
+    #define SPHERE_VER_FILEOS                0x0L    // VOS_UNKNOWN
```

Comment:
Can you consider the possibility of a 32 bits build on Mac?

Reply:
https://www.infoq.com/news/2018/01/macos-32bit-app-deprecation/ 32 bit build is Deprecate by Apple

Reply:
It's deprecated, but it's still possible if someone is motivated enough, right? To cover every possible case i'd like to add also 32 bits and even arm arch